packages/wrangler/src/deploy/index.ts

```diff
@@ -361,6 +363,15 @@ export async function deployHandler(
 		experimentalVersions: args.experimentalVersions,
 	});
 
+	if (deploymentId) {
```

Comment:
https://github.com/cloudflare/workers-sdk/pull/6383/files#r1705821768

Reply:
Ahah. So in that case I made the entry allow `worker_id` to be undefined!

Reply:
I can do that too for version uploads I guess.